### PR TITLE
Port test to ts

### LINKED ISSUE #1152: Port tests to typescript
### Expected behavior
Taiko tests should be run in both javascript and typescript to help check the typescript interface.

### Versions
```
Version: 1.0.6 (Chromium: 81.0.3994.0) RELEASE
```

### PR DESCRIPTION
**Pull request by NivedhaSenthil**
fixes #1152

### CODE REVIEW

File: test/functional-tests/tests/alert.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    scenarioStore.put('alert-text', true);
+  });
+}
+
+@Step('Check if alert was accepted')
+public checkAlertAccepted() {
+  assert.ok(scenarioStore.get('alert-text'));
+}
+
+}
```
Comment (negiDharmendra) [inline test/functional-tests/tests/alert.ts:20]: Please fix indentation.

File: test/functional-tests/tests/assert.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+@Step('Assert page has set timezome')
+public async assertPageHasSetTimezone() {
+  const getTime = await evaluate(() => {
+    return new Date(1479579154987).toString();
+  });
+  assert.equal(getTime, 'Sat Nov 19 2016 13:12:34 GMT-0500 (Eastern Standard Time)');
+}
+
+}
```
Comment (negiDharmendra) [inline test/functional-tests/tests/assert.ts:56]: Please fix indentation.

File: test/functional-tests/tests/alert.ts (hunk 1 above)
Comment (NivedhaSenthil) [reply test/functional-tests/tests/alert.ts]: Done

File: test/functional-tests/tests/assert.ts (hunk 2 above)
Comment (NivedhaSenthil) [reply test/functional-tests/tests/assert.ts]: Done

**Comment by NivedhaSenthil** [comment]
Do we really need to have FTs in both JS and TS ? 
As our unit tests are still written in JS covering most scenarios,
also I believe anything that works with typescript should also work with JS since we have only bindings, please correct me if I am wrong.

**Pull request merged**